[2025-01-10T17:22:04.850+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-10T17:22:04.865+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_information.append_to_database scheduled__2025-01-10T17:21:00+00:00 [queued]>
[2025-01-10T17:22:04.873+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_information.append_to_database scheduled__2025-01-10T17:21:00+00:00 [queued]>
[2025-01-10T17:22:04.875+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-10T17:22:04.889+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): append_to_database> on 2025-01-10 17:21:00+00:00
[2025-01-10T17:22:04.896+0000] {standard_task_runner.py:72} INFO - Started process 1457 to run task
[2025-01-10T17:22:04.899+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'stock_information', 'append_to_database', 'scheduled__2025-01-10T17:21:00+00:00', '--job-id', '157', '--raw', '--subdir', 'DAGS_FOLDER/vnstock_stream.py', '--cfg-path', '/tmp/tmpiiwkttqd']
[2025-01-10T17:22:04.903+0000] {standard_task_runner.py:105} INFO - Job 157: Subtask append_to_database
[2025-01-10T17:22:04.958+0000] {task_command.py:467} INFO - Running <TaskInstance: stock_information.append_to_database scheduled__2025-01-10T17:21:00+00:00 [running]> on host 453ac4defedc
[2025-01-10T17:22:05.049+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='TuanDz' AIRFLOW_CTX_DAG_ID='stock_information' AIRFLOW_CTX_TASK_ID='append_to_database' AIRFLOW_CTX_EXECUTION_DATE='2025-01-10T17:21:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-10T17:21:00+00:00'
[2025-01-10T17:22:05.052+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-10T17:22:20.833+0000] {logging_mixin.py:190} INFO - Lỗi kết nối db: ('HYT00', '[HYT00] [Microsoft][ODBC Driver 18 for SQL Server]Login timeout expired (0) (SQLDriverConnect)')
[2025-01-10T17:22:20.836+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-10T17:22:20.847+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-10T17:22:20.850+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=stock_information, task_id=append_to_database, run_id=scheduled__2025-01-10T17:21:00+00:00, execution_date=20250110T172100, start_date=20250110T172204, end_date=20250110T172220
[2025-01-10T17:22:20.878+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-10T17:22:20.893+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-10T17:22:20.899+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
